config/v1/tests/nodes.config.openshift.io/MinimumKubeletVersion.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: apiextensions.k8s.io/v1 # Hack because controller-gen complains if we don't have this
+name: "Node"
+crdName: nodes.config.openshift.io
+featureGate: MinimumKubeletVersion
+tests:
+  onCreate:
+    - name: Should be able to create a minimal Node
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec: {} # No spec is required for a Node
+      expected: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec: {}
+    - name: Should be able to create a minimumKubeletVersion
+      initial: |
+        apiVersion: config.openshift.io/v1
+        kind: Node
+        spec:
+          minimumKubeletVersion: 1.30.0
```

Comment:
Please add a test that shows the empty string passes validation

Reply:
done!